universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -110,12 +95,11 @@ public Tag renderProgramListItem(ProgramDefinition program) {
   }
 
   Tag renderEditLink(String text, long programId) {
-    return a(text)
-        .withHref(controllers.admin.routes.AdminProgramController.edit(programId).url())
-        .withClasses(
-            Styles.MR_2,
-            Styles.TEXT_BLUE_400,
-            StyleUtils.hover(Styles.TEXT_BLUE_500),
-            StyleUtils.applyUtilityClass(StyleUtils.RESPONSIVE_MD, Styles.MR_4));
+    String editLink = controllers.admin.routes.AdminProgramController.edit(programId).url();
+    return renderLink(
+        text,
+        editLink,
+        String.join(
+            " ", Styles.MR_2, StyleUtils.applyUtilityClass(StyleUtils.RESPONSIVE_MD, Styles.MR_4)));
```

Comment:
Suggestion (non-blocking): we end up doing a lot of `String.join(" ", yada, yada, ...);` maybe factor out `String joinClassNames(String ...classNames)` into `BaseHtmlView` for convenience?

Reply:
`StyleUtils` seems like a good place for that.

Reply:
Done.